File: lib/class/account.dart
class Account {

  int _id = 0;
  String _name = "";
  String _email = "";
  String _password = "";
  String _username = "";
  int _active = 1;
  int _type = 1;

  Account();

  Account.fromJson(Map<String, dynamic> json):
  _id = json['id'],
  _name = json['name'],
  _email = json['email'],
  _password = json['password'],
  _username = json['username'],
  _active = json['active'],
  _type = json['type']
  ;

  int get id => this._id;
  set id(int value) => this._id = value;

  get name => this._name;
  set name( value) => this._name = value;

  get email => this._email;
  set email( value) => this._email = value;

  get password => this._password;
  set password( value) => this._password = value;

  get username => this._username;
  set username( value) => this._username = value;

  get active => this._active;
  set active( value) => this._active = value;

  get type => this._type;
  set type( value) => this._type = value;


}

File: demoBiBlioCamp/src/main/java/com/example/demoBiBlioCamp/model/Account.java
package com.example.demoBiBlioCamp.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "BBC_Account")
public class Account {

    @Id
    private int id;

    @Column
    private String name;

    @Column
    private String email;

    @Column
    private String password;

    @Column
    private String username;

    @Column
    private int active;

    @Column
    private int type;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public int getActive() {
        return active;
    }

    public void setActive(int active) {
        this.active = active;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }
}
